yarn.lock

```diff
@@ -1789,6 +1789,11 @@
   dependencies:
     "@babel/types" "^7.3.0"
 
+"@types/emoji-mart@^3.0.5":
```

Comment:
You shouldn't commit your lock file if no changes were made to the package dependencies.